integration_tests/src/main/python/conditionals_test.py

```diff
@@ -28,7 +28,9 @@ def test_if_else(data_gen):
     null_lit = get_null_lit_string(data_gen.data_type)
     assert_gpu_and_cpu_are_equal_collect(
             lambda spark : three_col_df(spark, boolean_gen, data_gen, data_gen).selectExpr(
-                # A literal predicate is not supported yet
+                'IF(TRUE, b, c)',
+                'IF(TRUE, {}, {})'.format(s1, null_lit),
+                'IF(FALSE, {}, {})'.format(s1, null_lit),
```

Comment:
Does it support a literal predicate? like IF (3>2, b, c)

Reply:
Yes